src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -88,6 +88,13 @@ export class NotebookLinkHandler {
 			 * We return the absolute path for the link so that it will get used in the as the href for the anchor HTML element
 			 * (in linkCalloutDialog document.execCommand('insertHTML') and therefore will call getLinkURL() with HTMLAnchorElement to then get the relative path
 			*/
+			// Need to encode URI here in order for user to click the proper encoded link in WYSIWYG
+			// skip encoding it if it's already encoded
+			let encodedLinkURL = this._link;
```

Comment:
This var isn't necessary, just use this._link directly (and is potentially confusing since it may not actually be encoded at this point - that's what the next line is checking...)

```js
if (this._isFile || this._link=== unescape(this._link)) {
     return encodeURI(this._link);
}
return this._link;
```